localstack/services/apigateway/invocations.py

```diff
@@ -348,14 +348,15 @@ def invoke_rest_api_integration_backend(invocation_context: ApiInvocationContext
     if (uri.startswith("arn:aws:apigateway:") and ":lambda:path" in uri) or uri.startswith(
         "arn:aws:lambda"
     ):
-        if integration_type in ["AWS", "AWS_PROXY"]:
+        if integration_type == "AWS_PROXY":
+            return LambdaProxyIntegration().invoke(invocation_context)
+        elif integration_type == "AWS":
             func_arn = uri
```

Comment:
nit: great to see that the logic for `AWS_PROXY` has been extracted into a `LambdaProxyIntegration` class 👍  - perhaps we could also move the logic for `AWS` integration into its own class? (in a future iteration...)

Reply:
Yep, exactly - I mentioned that in the PR description, that is coming soon in the next integrations.